cookbooks/swift/recipes/setup.rb

```diff
@@ -88,7 +88,6 @@
 unrequired_packages = [
   "python-requests",  "python-six", "python-urllib3",
   "python-pbr", "python-pip",
-  "python3-requests",  "python3-six", "python3-urllib3",
```

Comment:
I had to review `/var/log/apt/history.log` to discover that this purge was also pulling out docker-compose